app/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentDownloadController.java

```diff
@@ -260,6 +256,19 @@ void microarrayExperimentDownload(String experimentAccession,
                                         experiment.getAccession(), ExperimentFileType.SUMMARY_PDF));
                         break;
 
+                    case PROTEOMICS_DIFFERENTIAL:
+                        paths.add(experimentFileLocationService.getFilePath(
+                                experiment.getAccession(), ExperimentFileType.CONDENSE_SDRF))
+                                .add(experimentFileLocationService.getFilePath(
+                                        experiment.getAccession(), ExperimentFileType.CONFIGURATION))
+                                .add(experimentFileLocationService.getFilePath(
+                                        experiment.getAccession(), ExperimentFileType.PROTEOMICS_RAW_QUANT))
```

Comment:
Is this file the one containing the log fold differential changes? What is the file name pattern of ExperimentFileType.PROTEOMICS_RAW_QUANT - based on the label only I would suspect that this refers to a pre contrast comparison (expression per sample) file.

Reply:
Yes, the raw file doesn't include the differential changes. I would change this file for a ExperimentFileType that adds into the zip the analytics file for proteomics differential (like `E-PROT-39-analytics.tsv`). You can keep the RAW quan as well, but the analytics file here is the most relevant one.